### PR TITLE
migrate to lit-html 2

### PR DESCRIPTION
**Pull request by taulinger**

### CODE REVIEW

File: src/utils/checks.js [hunk 1]
```diff
@@ -43,3 +52,12 @@ export const isPromise = (val) => {
 	// eslint-disable-next-line promise/prefer-await-to-then
 	return Boolean(val && typeof val.then === 'function');
 };
+
+/**
+ * Checks if a value is a lit-html TemplateResult.
+ * @param {*} val
+ * @returns boolean} true if it is a TemplateResult
```
Comment (thiloSchlemmer) [inline src/utils/checks.js:59]: Incomplete doc

**Review by thiloSchlemmer — DISMISSED** [review]
Well done!

File: src/utils/checks.js (hunk 1 above)
Comment (taulinger) [reply src/utils/checks.js:59]: thx

**Review by thiloSchlemmer — APPROVED** [review]
Thx!

**Pull request merged**